go.mod

```diff
@@ -1,37 +1,37 @@
 module github.com/bitrise-steplib/bitrise-step-manage-ios-code-signing
 
-go 1.21
-
-toolchain go1.23.2
+go 1.22
```

Comment:
`go–xcode` is now on 1.22 so this must be bumped.

Reply:
Yes, but this ship has already sailed unfortunately, and looks like one step release was already shipped with Go 1.22: https://github.com/bitrise-io/go-xcode/pull/258 FYI @godrei @naveen-bitrise

Either we got lucky and remaining users of Xcode 15 don't care or everyone upgraded their stacks.